src/core/events/bpf.rs

```diff
@@ -426,6 +504,26 @@ impl EventFactory for BpfEventsFactory {
     }
 }
 
+/// Size of data for a single log event. Please keep in sync with its
+/// BPF counterpart.
+pub(super) const BPF_LOG_MAX: usize = 128;
+
+/// Max number of log events we can store at once in the shared map. Please keep in
+/// sync with its BPF counterpart.
+pub(super) const BPF_LOG_EVENTS_MAX: u32 = 32;
+
+event_byte_array!(BpfLogData, BPF_LOG_MAX - 1);
```

Comment:
Why `- 1`? Is it to align the log even taking into account the level? If so, `BPF_LOG_MAX` should include that -1.

Same in the BPF side.

Reply:
don't recall the exact reason I made I left it out. But I don't see any to keep it.